### PR TITLE
Jwt authentication

### PR DESCRIPTION
**Pull request by diegoportela99**
Authentication for users

### CODE REVIEW

File: HELPS/Controllers/UserController.cs
Comment (ghost) [inline HELPS/Controllers/UserController.cs]: Wrong response code, should be 401

File: HELPS/Controllers/UserController.cs
Comment (ghost) [inline HELPS/Controllers/UserController.cs]: Returns invalid object (see spec), Token should be renamed to AccessToken and should include isAdmin

File: HELPS/Startup.cs
Comment (ghost) [inline HELPS/Startup.cs:71]: Missing UseAuthentication()?

File: HELPS/Controllers/UserController.cs
Comment (ghost) [inline HELPS/Controllers/UserController.cs:23]: This endpoint does not match spec, should be api/login

File: HELPS/Controllers/UserController.cs
Comment (ghost) [inline HELPS/Controllers/UserController.cs:38]: I am assuming this is for testing but definitely should not be included in production

**Review by ghost — CHANGES_REQUESTED** [review]
I did some very surface level testing and found a few issues so take this review with a grain of salt. 

I would suggest fixing the endpoints and protecting the user controller before merging (definitely before merging into develop) but you can merge this as is if you would prefer.

File: HELPS/Startup.cs
Comment (diegoportela99) [inline HELPS/Startup.cs:70]: Is this all that's needed? i'm not too sure

**Pull request merged**
